refinery/data_set_manager/search_indexes.py

```diff
@@ -140,7 +141,7 @@ def prepare(self, object):
         data.update({
             NodeIndex.DOWNLOAD_URL:
                 '' if file_store_item is None
-                else file_store_item.get_datafile_url(),
+                else get_full_url(file_store_item.get_datafile_url()),
```

Comment:
It's probably better to modify `get_datafile_url()` to return an absolute URL for local files. This is how it appears to be used throughout the codebase anyway (wrapped in `get_full_url()`).